src/SwedbankPay.Sdk/PaymentInstruments/Card/ICardResource.cs

```diff
@@ -1,25 +1,25 @@
-﻿using System;
-using System.Threading.Tasks;
-
-namespace SwedbankPay.Sdk.Payments.CardPayments
-{
-    public interface ICardPaymentsResource
-    {
-        /// <summary>
-        ///     Creates a new credit card payment
-        /// </summary>
-        /// <param name="paymentRequest"></param>
-        /// <param name="paymentExpand"></param>
-        /// <returns></returns>
-        Task<CardPayment> Create(CardPaymentRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.None);
-
-
-        ///// <summary>
-        /////     Gets an existing card payment.
-        ///// </summary>
-        ///// <param name="id"></param>
-        ///// <param name="paymentExpand"></param>
-        ///// <returns></returns>
-        Task<CardPayment> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None);
-    }
-}
+﻿using System;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.PaymentInstruments.Card
+{
+    public interface ICardResource
+    {
+        /// <summary>
+        ///     Creates a new credit card payment
+        /// </summary>
+        /// <param name="paymentRequest"></param>
+        /// <param name="paymentExpand"></param>
+        /// <returns></returns>
+        Task<ICardPaymentResponse> Create(CardPaymentRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.None);
+
+
+        ///// <summary>
+        /////     Gets an existing card payment.
+        ///// </summary>
+        ///// <param name="id"></param>
+        ///// <param name="paymentExpand"></param>
+        ///// <returns></returns>
+        Task<ICardPaymentResponse> Get(Uri id, PaymentExpand paymentExpand = PaymentExpand.None);

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)